.github/workflows/bump-version.yml

```
@@ -0,0 +1,294 @@
+          BUMP_TYPE_UPPER=$(echo $BUMP_TYPE | tr '[:lower:]' '[:upper:]')
+          if [ -n "$AI_PROVIDER" ]; then
+            AI_PROVIDER_TEXT=" (using $AI_PROVIDER)"
+          else
+            AI_PROVIDER_TEXT=""
+          fi
+
+          # Generate PR body
+          cat > /tmp/pr_body.md << 'EOF'
+## Version Bump: v${CURRENT_VERSION} -> v${NEW_VERSION}
+
+This PR bumps the version from **v${CURRENT_VERSION}** to **v${NEW_VERSION}** (${BUMP_TYPE_UPPER} bump).
```

actionlint (1.7.8)

could not parse as YAML: could not find expected ':'

(syntax-check)